layers/+distribution/spacemacs/packages.el

```diff
@@ -969,17 +965,15 @@ For instance pass En as source for English."
         (setq google-translate-default-source-language (downcase source))
         (setq google-translate-default-target-language (downcase target)))
       (spacemacs/set-leader-keys
+        "xgl" 'spacemacs/set-google-translate-languages
         "xgQ" 'google-translate-query-translate-reverse
         "xgq" 'google-translate-query-translate
         "xgT" 'google-translate-at-point-reverse
         "xgt" 'google-translate-at-point))
-    :config
-    (progn
-      (require 'google-translate-default-ui)
       (setq google-translate-enable-ido-completion t)
       (setq google-translate-show-phonetic t)
       (setq google-translate-default-source-language "en")
-      (setq google-translate-default-target-language "fr"))))
+      (setq google-translate-default-target-language "fr")))
```

Comment:
I like this "default" personally :stuck_out_tongue: , but wouldn't it be nice to introduce a `dotspacemacs-default-language` and set it as the default `target` language?